packages/shared/src/variants/freeze.ts

```diff
@@ -41,7 +41,10 @@ export class FreezeGo extends Baduk {
     return { ...super.exportState(), frozen: this.frozen };
   }
 
-  static uiTransform(config: NewBadukConfig, state: FreezeGoState) {
+  static uiTransform<ConfigT extends NewBadukConfig = NewBadukConfig>(
```

Comment:
Had to change this because TypeScript wants types to match the parent class even for `static` methods. It's strange haha